profile/src/v100/profile.rs

```diff
@@ -0,0 +1,701 @@
+use crate::prelude::*;
+/// Representation of the Radix Wallet, contains a list of
+/// users Accounts, Personas, Authorized Dapps per network
+/// the user has used. It also contains all FactorSources,
+/// FactorInstances and wallet App preferences.
```

Comment:
FIXME: Add more docs, and doc tests, this type alongside `Wallet` is the most important model in the whole repo.